run_dir/static/js/deliveries.js

```diff
@@ -295,7 +226,7 @@ $('.button-save-bioinfo-responsible').click(function() {
                 var old_number_of_projects = parseInt($(old_number_span).text().trim());
                 // hide if it was the last project
                 if (old_number_of_projects == 1) {
-                    $(old_checkbox).closest('div.form-group').hide();
+                    $(old_checkbox).closest('div.chkbox').hide();
```

Comment:
Just checking that `chkbox` isn't a typo?

Reply:
No, its a class name... Could have been better named, I guess.

Reply:
No worries, I guess its because checkbox is a html tag so that would not work. This is fine